lib/validate_encrypt_utils.pm

```diff
@@ -80,19 +141,29 @@ sub verify_cryptsetup_properties {
     }
 }
 
+=head2 verify_restoring_luks_backups 
+
+  verify_restoring_luks_backups($mapped_dev, $mapped_dev_path, $backup_file_info, $backup_path);
```

Comment:
Strictly speaking is should be `verify_restoring_luks_backups(%args)` and then you can describe the keys.

You also can show that those are key pairs, so it's clear that hash is passed to the method.